src/components/ServiceCard.jsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import {
+  Card,
+  CardContent,
+  CardFooter
+} from "@baltimorecounty/dotgov-components";
+
+let starImg = {
+  fontSize: ".2em",
+  float: "right",
+  marginTop: "0px",
+  width: "10px"
+};
+
+const ServiceCard = ({ name, department, url, icon, rank }) => (
+  <div className="d-flex col-lg-4 col-md-6 col-sm-6">
```

Comment:
this may seem minor but i would leave this out of the service card, and ratehr put it in teh render item. That way this component is not reliant on the bootstrap grid.

Reply:
so this would go into the ServiceList component? We wouldnt want to add this to the FilterList since that's supposed to stay generic correct?

Reply:
also, this is showing three cards per column on desktop, should be four.

Reply:
100% tim, inside of the renderItem function would be the bootstrap div, then the service card.

Reply:
done